angr/knowledge_plugins/labels.py

```diff
@@ -11,14 +11,15 @@ def __init__(self, kb):
         self._labels = {}
         self._reverse_labels = {}
         for obj in kb._project.loader.all_objects:
-            for v in obj.symbols:
-                if v.name and not v.is_import and v.type not in {cle.SymbolType.TYPE_OTHER, }:
-                    self._labels[v.rebased_addr] = v.name
-                    self._reverse_labels[v.name] = v.rebased_addr
             try:
+                for v in obj.symbols:
+                    if v.name and not v.is_import and v.type not in {cle.Symbol.TYPE_OTHER, }:
+                        self._labels[v.rebased_addr] = v.name
+                        self._reverse_labels[v.name] = v.rebased_addr
```

Comment:
What part of this will explode?

Reply:
weird, I moved the `try` back to where it was before and the issue I had disappeared...
well, if I will ever reproduce the issue I will get back to this.